.pre-commit-config.yaml

```diff
@@ -43,27 +43,31 @@ repos:
           - prettier-plugin-toml
           - prettier-plugin-sort-json
 
-  - repo: https://github.com/psf/black
```

Comment:
This ordering is consistent with other repos, it will need to be changed in the others as well. (all but nav, lint, compat) LMK I can do it.

Reply:
Do not hurry to do it. We might make other changes too. Still, I considered important to ensure someone else knows what/why.